README.md

```diff
@@ -2358,33 +2351,39 @@ Docker image:
 
 ### Cross-compilation
 
-Users have been reporting some degree of success using cross compiles
-from Linux x86 host to embedded ARM Linux devices.
+Cross-compilation is possible when using the CMake build system. First you have
+to build and install flatcc for the build architecture. Then when cross-compiling
+provide the install location of the build architecture flatcc package via
+environment variable `flatccCli_ROOT` or via cmake variable `flatccCli_ROOT`.
+
```

Comment:
Can we have all uppercase ENV variables here?

Reply:
This is a CMake feature for overriding locations of cmake modules.
See https://cmake.org/cmake/help/latest/command/find_package.html#id5

These should be changed to `flatcccli_ROOT` btw.